content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,83 @@
+
+#### Slightly Advanced (Free/Open Source):
+ - `OBS Studio (Windows, Mac, Linux)`: OBS Studio is free and open-source, offering more control over your recording, including screen capture, webcam input, and basic audio mixing. While powerful, it might have a small learning curve.
+ - `VLC Media Player (Windows, Mac, Linux)`: VLC can also be used for basic recording. It's a versatile player that can capture video from your webcam. 
+
+#### User Friendly (Paid):
+ - `Canva`: offers a user-friendly platform with recording capabilities. You can easily create slides with your key points and record yourself presenting them. Canva also provides editing features to polish your video.
+
+## How to Structure Your Video
+
+Here's a simple way to approach your **one-minute** video:
+
+#### 1. Start with the Basics
+
+Introduce yourself and your idea. Pull from the abstract of your talk and expand slightly to explain what your talk is about.
+
+#### 2. Share Why It Matters
+
+Why is your idea important to you? Why does it excite you? People love to see genuine enthusiasm - it is contagious! Highlight why your topic could be interesting or helpful to the audience.
+
+#### 3. Keep It Conversational
+
+Talk as if you are **explaining to a friend,** not reading from a script. Keep it engaging and natural.
+
+#### 4. Speak Clearly & With Energy
+
+Passion is contagious! **Slow down a bit,** use your voice expressively, and **have fun with it.**
+
+#### 5. Don't Worry About Perfection
+
+Your video doesn't need to be flawless - just **authentic**. If selected, you will get speaker training!
+
+#### 6. Humor and Creativity are Welcome
+
+If your idea lends itself to something quirky or funny, don't shy away. Even if your topic seems niche, an engaging presentation can make it appealing to everyone.  But don't worry if you are not naturally funny either - this is only optional.
+
+## **Tips for Success**
+
+* **Pep Talk Yourself:** Before recording, remind yourself that you are great at what you do. Boosting your confidence will help you project positivity and ease.  
+* **Keep It Simple:** Your video doesn't need to be polished or perfect. Focus on your enthusiasm and message.  
+* **Check Out Examples:** Watch [other video abstracts](https://posit.co/blog/how-we-build-the-positconf-program/) for inspiration.
+
+## **Need Feedback?**
+
+If you would like input on your abstract or video, we are happy to help! Share a link to your draft (Google Drive or similar), and we will assign a reviewer.
+
+- **Abstract** : Paste your abstract in a Google Doc/other sharable medium for review.
+
+- **Video Draft** : Share a link to your draft recording. The recording could be uploaded to Google Drive, YouTube (unlisted), Vimeo, or any other sharable platform.
+
+If you want feedback on specific aspects of your video (e.g., pacing, clarity, or technical quality), please mention it in the form. Otherwise, we will assume you’re looking for general feedback.
```

Comment:
```suggestion
By "video draft" we mean an initial recording that reviewers will offer one round of feedback on.  If you want feedback on specific aspects of your video (e.g., pacing, clarity, or technical quality), please mention it in the form. Otherwise, we will assume you’re looking for general feedback.
```